srcpkgs/telegram-desktop/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'telegram-desktop'
 pkgname=telegram-desktop
-version=4.1.1
-revision=1
+version=4.2.4
+revision=2
```

Comment:
```suggestion
revision=1
```
revision should only be incremented if the package keeps the same version and needs to be rebuilt